README.md

```diff
@@ -4,6 +4,22 @@ A webstore project built using the Spring Boot framework
 # Spring Boot framework Official Documentation
 https://docs.spring.io/spring-boot/docs/current-SNAPSHOT/reference/html/
 
+# Cloning Webstore to IntelliJ on Windows
+
+IntelliJ Git setup:
+- To connect Git to IntelliJ enter `Ctrl Alt s` > `version control` > `Git`
```

Comment:
maybe mention that the shortcut will open Settings modal

Reply:
Can do.